cookbooks/aws-parallelcluster-slurm/files/default/head_node_slurm/slurm/templates/slurm_parallelcluster_cgroup.conf

```diff
@@ -0,0 +1,9 @@
+# slurm_parallelcluster.conf is managed by the pcluster processes.
+# Do not modify.
+# Please add user-specific slurm configuration options in cgroup.conf
+
+{% if scaling_config.EnableMemoryBasedScheduling %}
+ConstrainRAMSpace=yes
+{% else %}
+ConstrainRAMSpace=no
```

Comment:
shall we leave the implicit default as we do today so that we are backward compatible and don't risk to break users that are already adding ConstrainRAMSpace in cgroup.conf as part of their customizations?